main.go

```diff
@@ -9,13 +9,13 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 
-	testutils "github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/test-utils"
-	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/utils"
+	testutils "github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/test-utils"
+	libUtils "github.com/hpe-hcss/hpegl-provider-lib/pkg/utils"
```

Comment:
this should be `github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/utils`